disnake/invite.py

```diff
@@ -175,6 +187,8 @@ def __init__(self, state: ConnectionState, data: InviteGuildPayload, id: int):
         self._icon: Optional[str] = data.get("icon")
         self._banner: Optional[str] = data.get("banner")
         self._splash: Optional[str] = data.get("splash")
+        self.nsfw_level: NSFWLevel = try_enum(NSFWLevel, data.get("nsfw_level", 0))
```

Comment:
`nsfw_level` doesn't seem to be `None`/missing in the payload, also, it seems to be 0 by default.
![image](https://user-images.githubusercontent.com/67214928/150659716-8d4ca18c-8f3c-4e64-b906-e5bedc5a2d39.png)